code/modules/reagents/reagent_containers/food/drinks/drinkingglass.dm

```diff
@@ -564,6 +564,24 @@
 					icon_state = "patronglass"
 					name = "Jack Rose"
 					desc = "Drinking this makes you feel like you belong in a luxury hotel bar during the 1920s."
+				if("alkasine")

```

Comment:
This, and the few lines below it should probably be changed to synthanol as well.